### ISSUE TITLE
Upgraded to  eslint@6.2.2 breaks when linting React files

### ISSUE THREAD

**Issue by ddaza** [mantoni/eslint_d.js#119]
Upgraded `eslint` and dependencies seems to have broken `eslint_d`. 
We have a combination of `.js` and `.ts(x)` files where `.js` can be either React or JS.

Currently using `eslint_d@8.0.0`

We also have some custom eslint rules.

Here are some of our dep: 
```
    "babel-eslint": "^10.0.3",
    "eslint": "6.2.x",
    "eslint-plugin-lodash": "^6.0.0",
```

I am seeing no problems with `eslint` linting but when using `eslint_d` I get this error.

```
$ yarn eslint_d --debug my-project/src/components/Video.js

Error: ENOENT: no such file or directory, open '/my-project/node_modules/acorn-jsx/index.js'
```

**Comment by mantoni** [comment]
To explain this: The process is called `eslint_d` which is why `killall node` didn't catch it. If you're using `eslint_d stop` it also cleans up the `~/.eslint_d` file with the PID and security secret.
